_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
+
+Now in a terminal, let's change our working directory to ``esmvaltool_tutorial``. Then, we run a text editor called Nano to open the configuration file:
+
+~~~bash
+  cd esmvaltool_tutorial
+  nano config-user.yml
 ~~~
-  cp config-user-example.yml config-user.yml
-~~~
-{: .source}
 
 This file contains the information for:
-  * Rootpaths to the data from different projects
-  * Directory structure for input data
-  * Number of available CPUs
-  * Destination directory
-  * Auxiliary data directory
-  * Output settings
-
-## Rootpaths to input data
-ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source, like CMIP for dataset from climate model intercomparison project, and OBS for observational dataset that adhere to (CMOR standard)[https://cmor.llnl.gov/].
-For each category, you can define either one path or several pathes as a list.
-In this lesson, you work with data from (CMIP5)[https://esgf-node.llnl.gov/projects/cmip5/].
-Add the root path of the folder where you downloaded the data during the (Setup)[https://escience-academy.github.io/lesson-esmvaltool/setup.html].
 
+* Rootpath to input data
+* Directory structure for the data from different projects
+* Number of parallel tasks
+* Destination directory
+* Auxiliary data directory
+* Output settings
+
+> ## Which text editor
+>
+> No matter what editor you use, you will need to know where it searches for and saves files. If you start it from the shell, it will (probably) use your current working directory as its default location. We use ``nano`` in examples because it is one of the least complex text editors. Press <kbd>ctrl</kbd> + <kbd>O</kbd> to save the file, and then <kbd>ctrl</kbd> + <kbd>X</kbd> to exit ``nano``.
+{: .callout}
+
+## Rootpath to input data
+
+ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source.
+The current categories in the configuration file are mentioned below. For example, CMIP is used for a dataset from the climate model intercomparison project whereas OBS for an observational dataset. The ``rootpath`` specifies the directories where ESMValTool will look for input data. For each category, you can define either one path or several paths as a list.
+
+~~~YAML
+rootpath:
+  CMIP3: [~/cmip3_inputpath1, ~/cmip3_inputpath2]
+  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+  CMIP6: [~/cmip6_inputpath1, ~/cmip6_inputpath2]
+  OBS: ~/obs_inputpath
+  OBS6: ~/obs6_inputpath
+  obs4mips: ~/obs4mips_inputpath
+  ana4mips: ~/ana4mips_inputpath
+  native6:  ~/native6_inputpath
+  RAWOBS: ~/rawobs_inputpath
+  default: ~/default_inputpath
 ~~~
+
+In this lesson, we will work with data from [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/).
+We add the root path of the folder where data is available.
```

Comment:
Maybe rephrase to "We add the root path of the folder where our/your data is available." I am not sure which one of the pronouns I would prefer, but I think it would clarify that the path to the student's data should be added here.